features/step_definitions/basic_steps.rb

```diff
@@ -36,5 +36,7 @@
   save_and_open_page
 end
 
-
+When("I select {string} from {string}") do |option, selector|
+  select(option, :from => selector)
```

Comment:
please stay consistent with the syntax, refactor this to poetry mode

Reply:
Done